litr-muxers/src/main/cpp/FFmpeg.h

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright 2022 LinkedIn Corporation
+ * All Rights Reserved.
+ *
+ * Licensed under the BSD 2-Clause License (the "License").  See License in the project root for
+ * license information.
+ *
+ * Author: Ian Bird
+ */
+
+#ifndef LITR_FFMPEG_H
+#define LITR_FFMPEG_H
+
+extern "C" {
```

Comment:
Because the code in this module will be C++, and libavformat (and deps) are C, I created this header to make it easy to import/include.